components/infra-proxy-service/service/service.go

```diff
@@ -21,14 +20,14 @@ type Service struct {
 	Logger      logger.Logger
 	ConnFactory *secureconn.Factory
 	Storage     storage.Storage
-	Authz       authz.SubjectPurgeClient
+	Authz       authz.AuthorizationClient
```

Comment:
The purge client was no longer being called anywhere. However, it looks like this top level Authz definition is also not being used anywhere. The database calling ValidateProjectAssignment has its own reference to the client so this can be dropped.